src/components/selection.tsx

```diff
@@ -42,7 +43,7 @@ export default class Selection extends React.Component<Props, {}> {
               resource={resource}
             />
 
-            <span className="ars-selection-button-icon" aria-hidden="true" />
+            <EditIcon />
```

Comment:
Does this remove `aria-hidden=true`? I wonder if this removes some accessibility support.

Reply:
Good question. I'm not clear if the empty span causes a verbal change, but it's easy enough to pass through this prop.

Reply:
Last I heard, an empty element is not read, making `aria-hidden` redundant in those cases.